### ISSUE TITLE
CustomElements V1

### ISSUE THREAD

**Issue by WebReflection** [WebPlatformTest/HTML5test#501]
Apparently there's only a V0 check like `'registerElement' in document` which would only score on Chrome.

The current V1 specs, agreed by all vendors, should be checked as `'customElements' in window` instead, or `"CustomElementsRegistry" in window`

Thanks.

**Comment by NielsLeenheer** [comment]
Should be fixed in the upcoming release of version 9.0

**Comment by WebReflection** [comment]
same as modules, this version 9.0 ... I'm not sure it's shipped. Still no Custom Elements support for WebKit/GNOME Web

**Comment by NielsLeenheer** [comment]
Version 9 was a 'bit' delayed while I had to focus on other projects. But the next version is back on track and should released before Christmas. 

The fixed test is currently available on https://alpha.html5test.com

**Comment by WebReflection** [comment]
I can confirm it says `yes` now, but I can also tell you that HTML import can be removed because it's been decided that it won't be implemented cross browser.

**Comment by WebReflection** [comment]
TL;DR **WebComponents** specification is:

  * Custom Elements V1
  * Shadow DOM
  * HTML templates

That's it

**Comment by WebReflection** [comment]
should I open a new ticket and close this?

**Comment by WebReflection** [comment]
no answers in a while ... closing this for now, please drop HTML imports from scores.
